draft-ietf-quic-qlog-main-schema.md

```diff
@@ -217,22 +236,26 @@ In order to make it easier to parse and identify qlog files and their
 serialization format, the "qlog_version" and "qlog_format" fields and their values
 SHOULD be in the first 256 characters/bytes of the resulting log file.
 
-An example of the qlog file's top-level structure is shown in {{top-element}}.
+An example of the qlog file's top-level structure is shown in {{qlog-file-def}}.
 
-~~~~~~~~
 Definition:
 
-class QlogFile {
-    qlog_version:string,
-    qlog_format?:string,
-    title?:string,
-    description?:string,
-    summary?: Summary,
-    traces: array<Trace|TraceError>
+~~~~~~~~
+QlogFile = {
+    qlog_version: text
+    ? qlog_format: text .default "JSON"
+    ? title: text
+    ? description: text
+    ? summary: Summary
+    ? traces: [+ Trace / TraceError]
 }
+~~~~~~~~
+{: .language-cddl}
```

Comment:
in #194 I have used `~~~~ cddl` to open the block instead of this line. 
It seems to generate the same HTML, it's a bit less verbose and it also is easier to parse with some simple awk so I have found it to be quite handy of a syntax compared to what you are using here.

It would be nice to keep things consistent.

Reply:
I agree your approach is probably the better option (and potentially makes it easier to extract CDDL from the markdown docs), so I'll change my side. Thanks!